src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -851,6 +861,8 @@ private async Task<T> ExecuteWithFailOverPolicyAsync<T>(IEnumerable<Configuratio
                 {
                     if (!success && backoffAllClients)
                     {
+                        _logger.LogWarning(LogHelper.BuildLastEndpointFailedMessage(_configClientManager.GetEndpointForClient(currentClient).ToString()));
```

Comment:
We dont need to get endpoint again - we can reuse previousEndpoint  or currentEndpoint